text/2018-09-17-schedule-limit.md

```diff
@@ -0,0 +1,114 @@
+# Summary
+
+This RFC proposes a new scheme to limit concurrent executing schedule operators. Compared to old implementation, it is more intuitive, easy to configure, and has better adaptability.
+
+# Motivation
+
+In order to prevent PD from generating a large number of schedule operators in a short period of time and affecting the performance of the tikv cluster, and to avoid the problem of over-balance, currently PD uses a limitation based scheme.
```

Comment:
```suggestion
In order to prevent PD from generating a large number of schedule operators in a short period of time and affecting the performance of the TiKV cluster, and to avoid the problem of over-balance, currently PD uses a limitation based scheme.
```